project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject datasplash "0.6.1"
+(defproject datasplash "0.6.2-SNAPSHOT"
```

Comment:
You might not want to merge this into `master`.

Reply:
Yes, sorry, corrected and added another pull request from right branches